vector/src/main/java/im/vector/app/features/settings/VectorSettingsPreferencesFragment.kt

```diff
@@ -103,7 +103,7 @@ class VectorSettingsPreferencesFragment :
         }
 
         findPreference<Preference>(VectorPreferences.SETTINGS_PREF_SPACE_CATEGORY)!!.let { pref ->
-            pref.isVisible = !vectorFeatures.isNewAppLayoutFeatureEnabled()
+            pref.isVisible = !vectorPreferences.isNewAppLayoutEnabled()
```

Comment:
Should we combine this with the previous condition? If the feature is disabled, we should not rely on the preference state. If the code is correct, the old layout should be displayed when the feature is disabled, even if the pref is true. 
```suggestion
            pref.isVisible = !vectorFeatures.isNewAppLayoutFeatureEnabled() || !vectorPreferences.isNewAppLayoutEnabled()
```

Reply:
I think this is not necessary. Looking at the implemention of `VectorPreferences.isNewAppLayoutEnabled()` there is already a check on the `vectorFeatures.isNewAppLayoutFeatureEnabled()`.